### PR TITLE
Guidelines added

### PR DESCRIPTION
**Pull request by gurjeetsinghvirdee**
### Closing: #3

### CODE REVIEW

**Review by diwash007 — CHANGES_REQUESTED** [review]
Thank you for your contribution, Gurjeet.
Here are some of the things you need to change on this:
1. No need to add screenshot images
2. Logos have been modified (please do not add files that you didn't change)
3. No mention of any way to set up the project locally
4. Invalid URL on add upstream
5. Change the file name to CONTRIBUTING.md

**Review by diwash007 — CHANGES_REQUESTED** [review]
1. The upstream link is still invalid
2.  The modified files haven't been untracked
3. To setup project, I mean how will a user would be able to test the extension on their computer not only git download .. but setting up the extension for use in the browser (chrome or edge)

**Comment by gurjeetsinghvirdee** [comment]
@diwash007 Images reverted same as before
and guidelines updated

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+### Never made an open source contribution before? Wondering how contributions work in our project? Here's a quick rundown!
+
+### Take a look at the Existing Issues or create your own Issues!
+- Wait for the Issue to be assigned to you after which you can start working on it.
+- Fork the Repo and create a Branch for any Issue that you are working upon.
+- Read the Code of Conduct
+- Create a Pull Request which will be promptly reviewed and suggestions would be added to improve it.
+
+## How to make a Pull Request
```
Comment (diwash007) [inline CONTRIBUTING.md:12]: remove this section

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+where `user-name` is your GitHub username. Here you're copying the contents of the PDF-Dark-Mode repository on GitHub to your computer.
+
+## Navigate to the project directory
+* After cloning the project in your computer, navigate to the project file using the command below.
+```
+cd PDF-Dark-Mode
+```
+
+## Add a reference(remote) to the original repository
```
Comment (diwash007) [inline CONTRIBUTING.md:58]: remove this section as well

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+git push -u origin <your_branch_name>
+```
+
+5. To create a pull request, click on `compare and pull request.
+
+6. Add an appropriate title and description to your PR explaining your changes.
+
+7. Cick on Create pull request. 
+
+## Setup the project in your Local Mahine:
```
Comment (diwash007) [inline CONTRIBUTING.md:118]: typo "Mahine" .... please set up the project on your machine and run it on your browser successfully and write the methods you followed to achieve that .... I have added that on README as well .. you can use that as reference

**Review by diwash007 — CHANGES_REQUESTED** [review]
I have commented on the different sections .. please update them

**Comment by gurjeetsinghvirdee** [comment]
> 1. The upstream link is still invalid
> 2. The modified files haven't been untracked
> 3. To setup project, I mean how will a user would be able to test the extension on their computer not only git download .. but setting up the extension for use in the browser (chrome or edge)

OK I will fix this

**Pull request closed without merging**